src/components/EventDashboard/ResourceTabPanel.tsx

```diff
@@ -0,0 +1,518 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useQuery, useMutation } from 'react-apollo';
+import {
+  Box,
+  Typography,
+  Button,
+  Checkbox,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TableSortLabel,
+  Toolbar,
+} from '@material-ui/core';
+import { FiberManualRecord, Add, Remove } from '@material-ui/icons';
+import { Order, stableSort, getComparator } from '../../utils/sort';
+import { GET_EVENT_BY_ID } from '../../graphql/queries/events';
```

Comment:
missed a linting error smh
`error  'GET_EVENT_BY_ID' is defined but never used  @typescript-eslint/no-unused-vars`

```suggestion
```